services/logger.js

```diff
@@ -15,33 +17,41 @@ const createLogger = () => {
   return createDevLogger(logLevel);
 };
 
-const createProdLogger = (logLevel) => {
+const createProdLogger = (currentLogLevel) => {
```

Comment:
no need to specify current, since it will never change.    

(if this were going to be used by different apps, it would be good to be generic)

Reply:
Ok, I will revert back.

Reply:
Done ✅